gamla/functional_generic.py

```diff
@@ -133,7 +133,20 @@ def lazyjuxt(*args, **kwargs):
     return lazyjuxt
 
 
-juxt = compose(after(tuple), lazyjuxt)
+def juxt(*funcs):
+    if _any_is_async(funcs):
+        funcs = tuple(map(after(to_awaitable), funcs))
+
+        async def inner_juxt_async(*args, **kwargs):
```

Comment:
the name is shown in all kinds of places (e.g. kg version keys), so the shorter `juxt` would be better

Reply:
ok, changed it to juxt and juxt_async